packages/shared/src/utils/constants/signup_fields.ts

```diff
@@ -0,0 +1,26 @@
+import { localize } from '@deriv/translations';
+
+export const address_details_fields = Object.freeze({
```

Comment:
Settings localized strings to const won't work when we change the language. Can we convert this to an fn?